### DISCUSSION TITLE
[Feature] 有没有办法让下载单个本子时候不建立一个名称为“1”的文件夹

### DISCUSSION THREAD

**Discussion by Arkyuyuzi** [hect0x7/JMComic-Crawler-Python#289] [Q&A] [answered]
### 新功能请求

很抱歉，没学过看不懂代码只能冒昧的提出要求![image](https://github.com/user-attachments/assets/64d46786-04dc-4765-be2c-fe7f12004bc9)

**Comment by hect0x7** [comment, marked as answer]
这个1应该表示的是章节序号，这是由下载文件夹规则控制的
你可以通过自定义option配置来改变这个规则：
```yml
dir_rule:
  rule: Bd / Ptitle # Bd表示根目录，Ptitle表示章节名称，意思是把图片下载到根目录下的一个以章节名称命名的文件夹
```
有关option配置可以参考文档→https://jmcomic.readthedocs.io/zh-cn/latest/option_file_syntax/
有关自定义文件夹名参考文档→https://jmcomic.readthedocs.io/zh-cn/latest/tutorial/9_custom_download_dir_name/

**Reply by hect0x7** [reply]
突然想到你这是使用GitHub actions下的吧，如果是的话，提交工作流的页面有个输入项就是自定义下载路径规则的，填那个再下载就行
